packages/common/src/types/userAttributes.ts

```diff
@@ -0,0 +1,29 @@
+export type OrgAttribute = {
```

Comment:
I'm not too sure about this naming, if you have suggestion for this model and the one below, let me know.

Reply:
seems fine to me